src/main/protobuf/heka.proto

```diff
@@ -37,7 +37,7 @@ message Field {
 message Message {
   required bytes    uuid        = 1;
   required int64    timestamp   = 2; // nanoseconds since UNIX epoch
-  optional string   type        = 3;
+  optional string   dtype       = 3;
```

Comment:
Why rename this to `dtype`?

Reply:
The code generator will make the fieldname `type`, which is a reserved keyword and breaks things.